apps/docs/src/pages/components/grid.mdx

```diff
@@ -0,0 +1,158 @@
+import { Callout } from "nextra/components";
+import { gridClasses, HvGrid } from "@hitachivantara/uikit-react-core";
+
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("Grid", "core", gridClasses);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Callout type="info">
+  If you want to create a simple grid where items are distributed accross a
+  specific number of columns, checkout the
+  [`HvSimpleGrid`](/components/simple-grid) component.
+</Callout>
+
+### Default grid
+
+The example below illustrates a common grid layout where items occupy different number of columns based on the screen size.
+
+```tsx live
+export default function Demo() {
+  const width = useWidth();
+  return (
+    <div className="flex flex-col gap-sm">
+      <HvTypography variant="label">{`Current width: ${width}`}</HvTypography>
+      <HvGrid container>
+        <HvGrid item xl={2} lg={3} md={4} sm={6} xs={12}>
+          <div className="flex items-center justify-center h-100px bg-atmo2">
+            xl=2 lg=3 md=4 sm=6 xs=1
```

Comment:
- the breakpoints in `md` don't fit the grid correctly.
- `xl={1}` is just too small. it should be at least `xl={2}`
- again, let's let `HvPanel` handle the styling. this (currently) doesn't work in dark-mode
- I think we should sort the breakpoints from smaller to larger. Smaller ones are more important and will be used more frequently.
- The breakpoint text doesn't match the props


![image](https://github.com/user-attachments/assets/af0332f1-c94c-4625-a689-836a24ff2e8e)



```suggestion
        <HvGrid item xs={12} sm={6} md={4} lg={3} xl={2}>
          <HvPanel className="content-center h-100px">
            xs=12 sm=6 md=4 lg=3 xl=2
```